pysrc/riffq/connection.py

```
@@ -89,7 +89,7 @@ def handle_auth(self, user: str, password: str, host: str, database: Optional[st
         """
         return callback(user == "user" and password == "secret")
 
-    def handle_connect(self, ip: str, port: int, callback: Callable[..., None] = lambda *a, **k: None) -> None:
+    def handle_connect(self, ip: str, port: int, server_name:str=None, callback: Callable[..., None] = lambda *a, **k: None) -> None:
```

Ruff (0.14.1)

Unused method argument: `ip`

(ARG002)